playbooks/install_stack.yaml

```diff
@@ -153,11 +153,38 @@
   - name: Install the tripleo client
     yum:
       name:
-      - ceph-ansible
       - python3-tripleoclient
     become: true
     become_user: root
 
+  - name: Prepare the deployment for Ceph
+    when: ceph_enabled
+    block:
+      - name: Check if ceph-ansible can be used to deploy Ceph
+        stat:
+          path: /usr/share/openstack-tripleo-heat-templates/environments/ceph-ansible
+        register: st_ceph
+      - name: Create ceph_ansible_enabled fact
+        set_fact:
+          ceph_ansible_enabled: "{{ st_ceph.stat.isdir is defined and st_ceph.stat.isdir | ternary(true, false) }}"
```

Comment:
nit: I'm pretty sure `| ternary(true, false)` is a noop and can just be omitted.